packages/amplify-category-auth/src/provider-utils/awscloudformation/import/index.ts

```diff
@@ -1232,7 +1231,7 @@ const headlessImport = async (
   // Import questions succeeded, create the create the required CLI resource state from the answers.
   const projectType: string = projectConfig.frontend;
 
-  const newState = await updateStateFiles(context, questionParameters, answers, projectType, false);
+  const newState = await updateStateFiles(context, questionParameters, answers, projectType, true);
```

Comment:
Why do we need this change of behavior?

Reply:
discussed offline. This is to maintain the consistency of updating the `team-provider-info.json` file between headless v/s non-headless mode